packages/slider/src/slider.css

```diff
@@ -137,3 +137,8 @@ governing permissions and limitations under the License.
     padding: 0;
     margin: 0;
 }
+
+:host([label-visibility='none']) #track {
+    grid-area: number;
```

Comment:
Is there a path to allowing this to work while not changing the grid area?

Reply:
yes. `align-self: center` seems to do the same thing